app/controllers/epic/EpicHoldbackTestsController.scala

```diff
@@ -0,0 +1,38 @@
+package controllers.epic
+
+import com.gu.googleauth.AuthAction
+import controllers.LockableS3ObjectController
+import models.EpicTests
+import play.api.libs.circe.Circe
+import play.api.libs.ws.WSClient
+import play.api.mvc._
+import services.FastlyPurger
+import services.S3Client.S3ObjectSettings
+import zio.DefaultRuntime
+
+import scala.concurrent.ExecutionContext
+
+object EpicHoldbackTestsController {
+  val name = "epic-holdback-tests"
+}
+
+class EpicHoldbackTestsController(
+  authAction: AuthAction[AnyContent],
+  components: ControllerComponents,
+  ws: WSClient, stage: String,
+  runtime: DefaultRuntime
+)(implicit ec: ExecutionContext) extends LockableS3ObjectController[EpicTests](
+    authAction,
+    components,
+    stage,
+    name = EpicHoldbackTestsController.name,
+    dataObjectSettings = S3ObjectSettings(
+      bucket = "gu-contributions-public",
+      key = s"epic/$stage/${EpicHoldbackTestsController.name}.json",
+      publicRead = true,  // This data will be requested by dotcom
+      cacheControl = Some("max-age=30"),
+      surrogateControl = Some("max-age=86400")  // Cache for a day, and use cache purging after updates
```

Comment:
now that we're fetching through the s3 api, we no longer need `publicRead`, `cacheControl`, `surrogateControl`

Reply:
ahhh nice of course! I guess we should do that for all the others too, but perhaps in a follow up?

Reply:
yep all the public/cache stuff can be removed in a separate PR

Reply:
I've pushed that up!